styles/responsive_styles.h

```diff
@@ -0,0 +1,68 @@
+#ifndef STYLES_RESPONSIVE_STYLES_H
+#define STYLES_RESPONSIVE_STYLES_H
+
+// Responsive Effect/Style Aliases by Fett263, Responsive Effects use BladeAngle and TwistAngle to control effects. Defaults are set to simplify implementation.
+
+// These Styles all use Layers<> over a BASE Color/Style
+
+// ResponsiveLockupL<LOCKUP COLOR, TRANSITION1, TRANSITION2, TOP, BOTTOM, SIZE>
+// Implements LocalizedLockup that will move based on the angle of the blade
+// TRANSITION1 & TRANSITION2 = transition Begin & End

```

Comment:
Should maybe TRANSITION1/2 default to TrInstant<> ir some short fade?

Reply:
That makes sense, TrInstant is probably the most logical, it would act like original Lockup